Documentation/project-docs/standard-platform.md

```diff
@@ -23,9 +23,9 @@ The .NET Platform Standard version represents binary portability across platform
 
 
 ## Terms
-- **.NET Platform Standard** - versioned sets of the reference assemblies that all .NET Platforms must support.
+- **.NET Platform Standard** - a specific versioned set of reference assemblies that all .NET Platforms must support as defined in the [CoreFX repo](https://github.com/dotnet/corefx).
```

Comment:
Problem is that that list is way larger than we are thinking for the Platform Standard. I'll take a stab at getting an updated list posted.